exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var name = "Rustumb";
+var nameLength = name.length;
+var myNameIs ="My name is ";
```

Comment:
Good to see your use of naming yor variables for other users to be able to know what's going on in your program.